benchmark/generate_diff.php

```diff
@@ -60,6 +61,24 @@ function formatDiff(?int $baseInstructions, int $headInstructions): string {
     return sprintf('%.2f%%', $instructionDiff / $baseInstructions * 100);
 }
 
+function find_benchmarked_commit_hash(string $commitHash): ?string {
```

Comment:
would be nice if $repo was passed from the callsite so the file path construction is identical.